org.mixedrealitytoolkit.uxcomponents/Slider/CanvasSlider.prefab

```diff
@@ -248,7 +251,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-  m_AnchoredPosition: {x: 0, y: 0}
+  m_AnchoredPosition: {x: -0.000017166138, y: -0.000002861023}
```

Comment:
Why this change?

Reply:
again, i changed nothing explicitly. Unity did this while reserializing

Reply:
Can you please revert these changes?

Reply:
See above comment (Unity did this during their serialization). I can _not_ update the prefabs, but that eliminates part of the benefit of this change.